Handlers/ReceiptsHandler.fs

```diff
@@ -0,0 +1,21 @@
+namespace ExpenseReports.Handlers
+
+open System.IO
+open Microsoft.AspNetCore.Http
+open Giraffe.HttpHandlers
+open Giraffe.Tasks
+
+module ReceiptHandler =
+    let receiptUploadHandler =
+        fun (next : HttpFunc) (ctx : HttpContext) ->
```

Comment:
Why the `let f = fun x -> something` construct? Why not just `let f x = something`?